docs/source/release_notes.rst

```diff
@@ -12,6 +12,7 @@ Release Notes
         * Added ``is_bounded_like_percentage`` property for objectives. If true, the ``calculate_percent_difference`` method will return the absolute difference rather than relative difference :pr:`1809`
         * Changed ``TargetEncoder`` to preserve custom indices in the data :pr:`1836`
         * Refactored ``explain_predictions`` and ``explain_predictions_best_worst`` to only compute features once for all rows that need to be explained :pr:`1843`
+        * Updated ``OutliersDataCheck`` implementation to calculate the probability of having no outliers :pr:`1745`
```

Comment:
Not sure what we are allowed to say given internals and where this impl came from 👀